ipc/cli/src/commands/checkpoint/mod.rs

```diff
@@ -35,7 +31,9 @@ pub(crate) struct CheckpointCommandsArgs {
 impl CheckpointCommandsArgs {
     pub async fn handle(&self, global: &GlobalArguments) -> anyhow::Result<()> {
         match &self.command {
-            Commands::ListBottomup(args) => ListBottomUpCheckpoints::handle(global, args).await,
+            Commands::ListBottomup(args) => {
+                LastBottomUpCheckpointHeight::handle(global, args).await
+            }
```

Comment:
What's with the duplication between `ListBottomup` and `LastBottomupCheckpointHeight`?

They call the same handler and accept the same arguments. Does it make sense to remove `LastBottomupCheckpointHeight` in favour of `ListBottomup`?

Reply:
oh yeah, probably need to remove.